src/components/TasksTable/index.tsx

```diff
@@ -172,9 +175,24 @@ const getColumnDefs = ({
       multiple: 4,
     },
     className: "cy-task-table-col-STATUS",
-    render: (status: string, { execution, id }) =>
-      status && (
-        <TaskStatusBadge status={status} id={id} execution={execution} />
+    render: (status: string, { dependsOn, execution, id }) =>
+      dependsOn?.length && status === TaskStatus.Blocked ? (
+        <Tooltip
```

Comment:
You may have to tweak the z-index (the table icons overlap it)!:

<img width="343" alt="Screenshot 2024-02-14 at 10 27 23 AM" src="https://github.com/evergreen-ci/spruce/assets/47064971/1b96c9a4-9ee3-4f89-ac49-9c6e9c379a53">